BrowserServicesKit/Sources/BrowserServicesKit/PrivacyConfig/Features/PrivacyFeature.swift

```diff
@@ -68,6 +68,9 @@ public enum PrivacyFeature: String {
     case tabManager
     case webViewStateRestoration
     case experimentalBrowserTheming
+    case macOSBrowserConfig
```

Comment:
These two platform-specific config features are based on Android's naming of the equivalent.